vyxal/elements.py

```diff
@@ -2976,9 +2976,10 @@ def gen():
             found = 0
             item = 1
             while True:
+                res = safe_apply(function, item, ctx=ctx)
```

Comment:
should this be called only after checking `found == count`, for maximum laziness?